tdnf.pc.in

```diff
@@ -6,6 +6,6 @@ includedir=@includedir@/tdnf
 Name: tdnf
 Description: tiny dandified yum
 Version: @VERSION@
-Requires:libsolv rpm
+Requires: libsolv libsolvext rpm libcurl
```

Comment:
It should be rpm-libs instead of rpm.

Reply:
The pkgconfig module is `rpm`. And I didn't even change that.